pkg/networkserver/redis/registry.go

```diff
@@ -75,7 +75,6 @@ func (r *DeviceRegistry) GetByID(ctx context.Context, appID ttnpb.ApplicationIde
 	}
 	return applyDeviceFieldMask(nil, pb, append(paths,
 		"ids.application_ids",
-		"ids.dev_addr",
```

Comment:
We still want to return the DevAddr to the user, right? (also below)

Reply:
I missed this one... why, indeed?

Reply:
If it is specified in the `get` fieldmask, yes.

Reply:
No, this should be implicit. `--ids.dev-addr` is not a valid flag in CLI.